ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact/cucushift-installer-rehearse-aws-ipi-compact-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-compact
```

Comment:
you don't need to define this workflow again, #36095 has been merged, have you rebased code from upstream to your local branch?

Reply:
I haven't rebased code.